android/build.gradle

```diff
@@ -39,7 +40,6 @@ allprojects {
         }
 
         google()
-        jcenter()
```

Comment:
jcenter is donezo: https://github.com/react-native-community/discussions-and-proposals/issues/331

Reply:
I made this change in Mobile-Expensify and was going to make it here too, but JCenter changed their mind and will support old packages forever.

Reply:
Regardless, we should stick with the upgrade guide and use mavencentral instead, don't you think?

Reply:
I think the update guide is wrong here, but I guess it's NAB 🤷‍♂️